components/panel/panel.module.css

```diff
@@ -2,6 +2,7 @@
   background-color: var(--mantine-color-gray-0);
   height: 100%;
   padding: 0 1rem;
+  overflow: scroll;
```

Comment:
panel contents getting too long not to scroll. there's a horizontal scroll right now too but only because the charts aren't fully styled -- will make sure that's not in the final iteration of the PR